sdk/rust/oak/src/lib.rs

```diff
@@ -31,26 +31,13 @@ pub mod rand;
 pub mod storage;
 
 pub mod proto {
-    pub mod google {
-        pub mod protobuf {
-            include!(concat!(env!("OUT_DIR"), "/google.protobuf.rs"));
-        }
-        pub mod rpc {
-            include!(concat!(env!("OUT_DIR"), "/google.rpc.rs"));
-        }
-    }
     pub mod oak {
+        use oak_abi::proto::oak::label;
```

Comment:
Maybe add a comment to clarify why this re-export is needed? Presumably we would like to make it so that developers only need to depend on `oak` and not `oak_abi` directly? I guess that makes sense, though I'm always a bit hesitant to introduce re-exports, as having multiple names for the same things can be confusing.

Reply:
Added a comment &ndash; the storage message definitions rely on the `Label` message, which is now (only) generated under the `oak_abi` crate.